### ISSUE TITLE
Landing Pages loading wrong directory on local installation. 

### ISSUE THREAD

**Issue by atwellpub** [inboundnow/landing-pages#230]
Reference:
https://github.com/inboundnow/landing-pages/issues/228#issuecomment-138230841

Summary:
Looks like local instance is trying to load countdown lander from the /wp-content/uploads/landing-pages/tempates directory instead of the core directory.

Code involved in making this selection:
https://github.com/inboundnow/landing-pages/blob/master/classes/class.landing-pages.php#L151-L156

@daprela I'm not sure how to trouble shoot this one due to it being specific to your server. Will you spend some time on it, confirm it is the issue, and maybe see if there is a simple solution?

**Comment by daprela** [comment]
For what I've been able to see this is what happens:
The variable `$template`
https://github.com/inboundnow/landing-pages/blob/master/classes/class.landing-pages.php#L138
should contain the base name of the template directory. Instead, that variable contains the absolute path to the template, which leads to the error of not finding the template in the following if statement.

I've gone deeply into this, and discovered that also the template option 'lp-selected-template' in the database should contain the base directory name, and instead it contains the absolute path. By changing manually the value of this field to the base name the template works again. 

Now, I'm trying to find out where that option field is set to track the root of the problem but there's no explicit assignment in the plugin, so if you could help me by telling me how and where you set the template options that would speed up my research quite a lot

**Comment by daprela** [comment]
The problem lies in the function `inbound_get_parent_directory()` and `lp_get_parent_directory()`.
Two problems
On top of the files config.php and index.php we call the function `inbound_get_parent_directory()`. That call is ignored by the system as if the function does not exists but strangely enough I don't get any fatal error like the one I reported with the issue #232 .

If I replace the call to `inbound_get_parent_directory` with `lp_get_parent_directory` then the function is called and executed but it returns the absolute path. The reason for this is that I have a local install with Apache ( `$_SERVER['SERVER_SOFTWARE']` == 'Apache') but my system is Windows, so the path contains '/' and not '\'.
I've fixed the issue in my system inside both functions lp_get_parent_directory() and inbound_get_parent_directory(), and now it returns the correct value but can I ask you why you don't use the function `basename(dirname(__FILE__))` to get the key? Why is the function `lp_get_parent_directory()` necessary?

**Comment by daprela** [comment]
I pushed my fixes to both functions, please take a look. In my opinion, the 'else' part of the function could now be the whole function

**Comment by atwellpub** [comment]
I wondered why we did not use basename(dirname(__FILE__)) as well, and as I saw how our lp_get_parent_directory method seemed to account for diffierent operating systems I got worried that there was a severe reason based on old troubleshooting tickets that we were using the custom method...

But there may not be. I may have not had known about basename(dirname(__FILE__)) at the time and the custom methods were my attempt to make due with what I understsood. 

If basename(dirname(__FILE__)) takes care of it we should use it. At no point in my deving or testing did I ever get the absolute paths instead of the folder name. 

Remember to add the issue URL to the relative commit so GitHub references the commit in the issue and we all can easily study your changes.
